libr/bin/format/elf/elf.c

```diff
@@ -784,7 +917,7 @@ static Sdb *store_versioninfo(struct Elf_(r_bin_elf_obj_t) *bin) {
 		}
 		if (bin->shdr[i].sh_offset + ((i + 1) * sizeof (Elf_(Shdr))) > bin->size) {
 			eprintf ("Warning: Too big version info field %d (%d)\n", i, size);
-			break;
+			//break;
```

Comment:
also the version info is not used at all, and having 9999999999 objects versioned in the elf header is totally useless imho